uninstall.sh

```diff
@@ -234,9 +234,9 @@ done
 # Attempt to locate Homebrew unless `--path` is passed
 if [[ "${#homebrew_prefix_candidates[@]}" -eq 0 ]]
 then
-  prefix="$(brew --prefix)"
+  prefix="$("${homebrew_prefix_default}"/bin/brew --prefix)"
```

Comment:
This is a little weird. If we're already assuming that Homebrew lives in `homebrew_prefix_default`, why do we need to query the `brew` that lives there with `--prefix`?

That said, I don't really follow the intent of this code fragment before the change, so I don't know what the correct thing to do here is.

Reply:
> If we're already assuming that Homebrew lives in `homebrew_prefix_default`, why do we need to query the `brew` that lives there with `--prefix`?

Because `homebrew_prefix_default` might be a symlink, despite its lack of effect w.r.t. Homebrew operations: https://github.com/Homebrew/install/issues/668